policyengine_api/endpoints/household.py

```diff
@@ -47,14 +48,29 @@ def add_yearly_variables(household, country_id):
                         if variables[variable]["isInputVariable"]:
                             household[entity_plural][entity][
                                 variables[variable]["name"]
-                            ] = {2023: variables[variable]["defaultValue"]}
+                            ] = {
+                                household_year: variables[variable][
+                                    "defaultValue"
+                                ]
+                            }
                         else:
                             household[entity_plural][entity][
                                 variables[variable]["name"]
-                            ] = {2023: None}
+                            ] = {household_year: None}
     return household
 
 
+def get_household_year(household):
+    """Given a household dict, get the household's year
+
+    Args:
+        household (dict): The household itself
+    """
+    household_year = list(household["people"]["you"]["age"].keys())[0]
```

Comment:
do we need the print statement?
```suggestion
    return list(household["people"]["you"]["age"].keys())[0]
```